File: OOP C++/Templete/Generic Function/Demo1.cpp
#include <iostream>
using namespace std;

template <typename Demo1>
void show(Demo1 a, Demo1 b)
{
    cout << "\nThe value of a & b is : " << a << "\t" << b;
}
int main()
{

    show(56, 48);
    show(10.14, 26.74);
    show("Imam", "Hasan");
    show('S', 'H');

    getchar();
    return 0;
}

File: OOP Java/Static/Static/Demo1.java

package Static;

public class Demo1 {
    String name;
    int id;
    static String var="CSTU";
    
    Demo1(String n,int i)
    {
        name=n;
        id=i;
    }
    void show()
    {
        System.out.println(name +"\t"+id +"\t"+var);
    }
}
